### ISSUE TITLE
[Bug]: Missing single quote (') in the Matplotlib documentation at the 'Quick start guide/Figure' section

### ISSUE THREAD

**Issue by tiandishenxian** [matplotlib/matplotlib#26305]
### Bug summary

I noticed a missing single quote (') in the Matplotlib documentation at the '[Quick start guide/Figure](https://matplotlib.org/3.7.1/tutorials/introductory/quick_start.html#a-simple-example)' section. There should be a single quotation mark after the right_bottom.

### Code for reproduction

```python
It should look something like this:
fig, axs = plt.subplot_mosaic([['left', 'right-top'],
                               ['left', 'right_bottom']])
```

### Actual outcome

![image](https://github.com/matplotlib/matplotlib/assets/60169323/58bdd4ed-37b9-4a65-9b4e-444ba1ecf12c)

![image](https://github.com/matplotlib/matplotlib/assets/60169323/f6c9b9e2-e3d4-40eb-85e9-c05a54cc9476)

### Expected outcome

![image](https://github.com/matplotlib/matplotlib/assets/60169323/390cd2e3-6794-42e3-8d92-47ac813be027)

### Matplotlib Version

3.7.1

### Installation

None

**Comment by tacaswell** [comment]
That is sourced from https://github.com/matplotlib/matplotlib/blob/6391678cc16042ceebb1dec1a6e2aa224ce77366/galleries/users_explain/quick_start.py#L66-L67

Would you be willing to open a PR fixing the typo @tiandishenxian ?

**Comment by tacaswell** [comment]
Thank you for reporting this @tiandishenxian ! 

It looks like this was fixed in https://github.com/matplotlib/matplotlib/pull/25519 and the correct version is published in the 3.7.2 docs.

https://matplotlib.org/3.7.2/tutorials/introductory/quick_start.html#a-simple-example and https://matplotlib.org/stable/tutorials/introductory/quick_start.html#a-simple-example have the corrected code.

**Comment by tiandishenxian** [comment]
> Thank you for reporting this @tiandishenxian !
> 
> It looks like this was fixed in #25519 and the correct version is published in the 3.7.2 docs.
> 
> https://matplotlib.org/3.7.2/tutorials/introductory/quick_start.html#a-simple-example and https://matplotlib.org/stable/tutorials/introductory/quick_start.html#a-simple-example have the corrected code.
Thanks for your reply, I didn't notice I was reading the outdated version! haha
